### PR TITLE
Fix commerce media servlet

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-10704

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/servlet/CommerceMediaServlet.java [hunk 1]
```diff
@@ -192,22 +193,16 @@ private long _getGroupId(
...
-				_commerceCatalogModelResourcePermission.check(
-					PermissionThreadLocal.getPermissionChecker(),
-					cpDefinition.getCommerceCatalog(), ActionKeys.VIEW);
-			}
-			else {
-				_commerceProductViewPermission.check(
-					PermissionThreadLocal.getPermissionChecker(),
-					commerceAccountId, cpDefinition.getCPDefinitionId());
-			}
+			_commerceProductViewPermission.check(
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/servlet/CommerceMediaServlet.java:203]: Keep the catalog permission here

**Review by alessiorendina — CHANGES_REQUESTED** [review]
Hey @andrea-ale-sbarra , please check my comment, thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Review by alessiorendina — APPROVED** [review]

**Comment by victoroscuro** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:stop

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**